Scripts/Run-SS-On-Exp&Sim/scripts/DataAnalysis.py

```diff
@@ -141,7 +142,7 @@ def func_current_bacteria_info(cs, dt):
 
 
 def micro_colony_analysis(pickle_files_directory, summary_statistic_method, dt, min_size_of_micro_colony,
-                          max_distance_between_cells, um_pixel_ratio):
+                          max_distance_between_cells, um_pixel_ratio, fig_path):
```

Comment:
fig_path should be an optional parameter; this will give us flexibility in choosing which summary stats we want to process.

After making fig_path an optional parameter, include an if statement:
if fig_path:
    ... define variables for jaggedness stuff...
    ...calculate jaggedness...